pkg/function/describe_backups.go

```diff
@@ -92,13 +92,21 @@ func describeBackupsPodFunc(cli kubernetes.Interface, tp param.TemplateParams, n
 			break
 		case strings.Contains(err.Error(), restic.PasswordIncorrect):
 			return map[string]interface{}{
+					DescribeBackupsSnapshotIDs:       nil,
```

Comment:
Can we updated the unit test in data_test.go to catch this issue please?

Reply:
In `data_test` we only `runAction`, so this was not caught.
When I ran this `action` against a cluster, it failed like this:
```
time="2019-10-11T03:38:26Z" level=info msg="Completed phase describeBackupsForRepo"
time="2019-10-11T03:38:26Z" level=error msg="Failed to render output artifacts template: config:1:10: executing \"config\" at <.Phases.describeBackupsForRepo.Output.fileCount>: map has no entry for key \"fileCount\"\nvendor/github.com/kanisterio/kanister/pkg/param.renderStringArg\n\t/root/src/github.com/kastenhq/k10/go/src/vendor/github.com/kanisterio/kanister/pkg/param/render.go:115\nvendor/github.com/kanisterio/kanister/pkg/param.RenderArtifacts\n\t/root/src/github.com/kastenhq/k10/go/src/vendor/github.com/kanisterio/kanister/pkg/param/render.go:94\nvendor/github.com/kanisterio/kanister/pkg/controller.(*Controller).runAction.func1\n\t/root/src/github.com/kastenhq/k10/go/src/vendor/github.com/kanisterio/kanister/pkg/controller/controller.go:447\nvendor/gopkg.in/tomb%2ev2.(*Tomb).run\n\t/root/src/github.com/kastenhq/k10/go/src/vendor/gopkg.in/tomb.v2/tomb.go:163\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1337"
time="2019-10-11T03:38:26Z" level=info msg="Updated ActionSet 'k10-describe-k10-dr-bp-8jn57-catalog-svc-kasten-io-deploympbtsp' Status->failed"
```

We need controller level tests to catch this